src/month.jsx

```diff
@@ -37,6 +38,15 @@ export default class Month extends React.Component {
     accessibleMode: PropTypes.bool
   };
 
+  constructor(props) {
+    super(props);
+
+    this.dayFormat = "MMMM D, YYYY.";
+    this.state = {
+      readInstructions: false
```

Comment:
Should `readInstructions` ever be toggled back to `false`?

Reply:
Sorry, I agree that it should start _false_. I'm thinking about when the user tabs to another part of the UI and then goes back, do we want that message re-read to them? As it is right now the region never went away or changed so there wouldn't be any notification, right?

Reply:
Works for me, just wanted to be sure :)

There's a `console.log` to remove from `time.jsx`, otherwise this all looks good.